gtfs/spec/en/reference.md

```diff
@@ -248,16 +250,16 @@ File: **Conditionally Required**
 
 File: **Conditionally Required**
 
-The [calendar_dates.txt](#calendar_datestxt) table can explicitly activate or disable service by date. It can be used in two ways.
+The [calendar_dates.txt](#calendar_datestxt) table explicitly activates or disables service by date. It can be used in two ways.
```

Comment:
Should be "It may be used in two ways". The implication is that these are the only two ways the file may be used. Using "can" implies even more uses.